server/service/service_campaign_test.go

```diff
@@ -0,0 +1,167 @@
+package service
+
+import (
+	"context"
+	"crypto/tls"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/WatchBeam/clock"
+	hostctx "github.com/fleetdm/fleet/v4/server/contexts/host"
+	"github.com/fleetdm/fleet/v4/server/contexts/viewer"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/fleetdm/fleet/v4/server/live_query"
+	"github.com/fleetdm/fleet/v4/server/mock"
+	"github.com/fleetdm/fleet/v4/server/ptr"
+	"github.com/fleetdm/fleet/v4/server/pubsub"
+	ws "github.com/fleetdm/fleet/v4/server/websocket"
+	kitlog "github.com/go-kit/kit/log"
+	"github.com/gorilla/websocket"
+	"github.com/stretchr/testify/require"
+)
+
+func TestService_StreamCampaignResults(t *testing.T) {
```

Comment:
Can you please add a comment indicating the purpose of this test? I feel the context will be lost once this PR is merged.